lms/static/sass/base/_base.scss

```diff
@@ -130,7 +130,6 @@ a:focus {
   margin: 0 auto 0;
   padding: ($baseline*2) 0;
   max-width: grid-width(12);
-  min-width: 760px;
   width: flex-grid(12);
```

Comment:
ASIDE: Should this be removed if we're relying on Neat now to define our grid? @AlasdairSwan, was there a reason why you didn't remove this when adding in the Neat breakpoints and configuration (like the ``max-width: grid-width(12)`` rule above this one)?